binventory/binventory/binventory.m

```diff
@@ -22,52 +22,21 @@ - (void)removeStatusLabelForType:(int)arg1;
 @implementation DKTrashTile
 
 - (void)dk_updateCount {
-    NSUInteger x = 0;
+    long x = 0;
```

Comment:
Never use primitive types like this when writing code for Darwin platforms using Apple SDKs. You should always opt for NSInteger/NSUInteger when you don't care about the width of the type.